### ISSUE TITLE
TIP: IF YOU GET `error: connection closed` - GO HERE

### ISSUE THREAD

**Issue by KittyLe** [discordjs/RPC#102]
Not an issue, just telling everyone. 

If you get `Error: connection closed`, make sure to use `ipc` as your transport. I have f***ed with this myself enough, so I'm here to tell it. 
You have to get your application whitelisted by Discord in order to use IPC now. If you have any applications from ~April 2019 - use them, they are whitelisted (at least mine was), since that restriction wasn't enforced back then. 
An app from June 2019 never worked for me, tho it might be working, I was lazy to test. 

TL;DR use an app from 2019 in order to get past this restriction OR ask a permission from Discord.

**Comment by isuckatprogram** [comment]
bruh

**Comment by isuckatprogram** [comment]
what if my account was made in 2020

**Comment by isuckatprogram** [comment]
i have to contact discord just to use discord-rpc

**Comment by advaith1** [comment]
this issue is false, it works fine with new applications

**Comment by KittyLe** [comment]
Never did for me, nor my friends, nor a few people there - scroll down in
issues tab.

– kittyle.

**Comment by KittyLe** [comment]
Also well, RPC doesn't just make a nice Rich Presence status, but it can do
a lot of other useful things, which from what I saw need an app to be
whitelisted.

– kittyle.

**Comment by geenva** [comment]
Confirming this is false, just tested with a Dec. 2020 client app.
